src/Hasher/Algorithm.php

```diff
@@ -0,0 +1,12 @@
+<?php
+
+declare(strict_types=1);
+
+namespace AltchaOrg\Altcha\Hasher;
+
+enum Algorithm: string
```

Comment:
An enum makes it a lot easier for users to discover all available options. They don't even have to be documented or validated when they're used (only when instantiating from input data).